fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -457,8 +457,14 @@ private long getTimestamp(final String property, final long created) throws Repo
         LOGGER.trace("Using {} date", property);
         final long timestamp = getProperty(property).getDate().getTimeInMillis();
         if (timestamp < created && created > NO_TIME && !isFrozenResource()) {
-            LOGGER.trace("Updating {} with later created date", property);
-            getNode().setProperty(property, created);
+            try {
+                LOGGER.trace("Updating {} with later created date", property);
+                getNode().setProperty(property, created);
+            } catch (final javax.jcr.AccessDeniedException e) {
+                LOGGER.trace("Unable to update {} with the later created date", property);
```

Comment:
The setProperty call is there to make sure that the created date is never later than the modified date (which often happens when creating nodes without this code, because we set the modified date, and then the created date is set a few milliseconds later when the node is saved).

Instead of updating the property, we could just return the later date, and accept that we'd be doing this every time.  What we're doing now (trying to update the property and burying the error if access is denied) effectively updates the property if the user has permissions to do so.

An alternative that I considered was doing a two-step process, where new nodes would be saved, and then we would manually set the created/modified times and re-save.  This has the advantage of making consistent dates possible from the start, at the cost of doing an extra save for each new node.